make/include/registry

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+set -o errexit -o nounset
+
+IMAGE_ORG=${IMAGE_ORG:-'splatform'}
+IMAGE_PREFIX=${IMAGE_PREFIX:-'uaa'}
+
+registry_state() {
+    if test -z ${IMAGE_REGISTRY+nil}; then
+        echo unset
+    else
+        if test -z ${IMAGE_REGISTRY:-''}; then
+            echo empty
+        else
+            echo set
+        fi
+    fi
+}
+
+case $(registry_state) in
+    set|unset)
+        IMAGE_REGISTRY=${IMAGE_REGISTRY%/}/
+        ;;
+    empty)
+        IMAGE_REGISTRY=''
+        ;;
+esac
```

Comment:
Lines 8 ~ 27 can be replaced with
```bash
IMAGE_REGISTRY="${IMAGE_REGISTRY:+${IMAGE_REGISTRY%/}/}"
```

(I'm pretty sure the SCF version already is)